tap_github/streams.py

```diff
@@ -22,30 +22,30 @@ def get_schema(catalog, stream_id):
     stream_catalog = [cat for cat in catalog if cat['tap_stream_id'] == stream_id ][0]
     return stream_catalog
 
-def get_child_full_url(child_object, repo_path, parent_id, grand_parent_id):
+def get_child_full_url(domain, child_object, repo_path, parent_id, grand_parent_id):
```

Comment:
declarations can be updated for parameter type hinting

`def get_child_full_url(domain :str, child_object :Any, repo_path :str, parent_id :int, grand_parent_id :int) -> str:

Reply:
as per discussion with @prijendev can be picked up later.